Sources/Controllers/TargetMenu/OATransportStopsBaseController.mm

```diff
@@ -189,7 +191,125 @@ - (void) processTransportStop
     self.nearbyRoutes = nearbyRoutes;
 }
 
-- (void) sortTransportStopsExits:(OsmAnd::LatLon)latLon stops:(NSMutableArray<OATransportStop *> *)stops
++ (OATransportStop *) findBestTransportStopForAmenity:(OAPOI *)amenity
```

Comment:
Best -> nearest?

Reply:
It's the same name like in Android.

https://github.com/osmandapp/Osmand/blob/bef177c5d7a415b747e377b7ff2cda844bb6c05f/OsmAnd/src/net/osmand/plus/mapcontextmenu/controllers/TransportStopController.java#L235

Reply:
In Android, they use a bad name)